platform/default/mbgl/util/default_styles.cpp

```diff
@@ -4,12 +4,12 @@ namespace mbgl {
 namespace util {
 namespace default_styles {
 
-const DefaultStyle streets          = { "mapbox://styles/mapbox/streets-v9",           "Streets" };
-const DefaultStyle outdoors         = { "mapbox://styles/mapbox/outdoors-v9",          "Outdoors" };
-const DefaultStyle light            = { "mapbox://styles/mapbox/light-v9",             "Light" };
-const DefaultStyle dark             = { "mapbox://styles/mapbox/dark-v9",              "Dark" };
-const DefaultStyle satellite        = { "mapbox://styles/mapbox/satellite-v9",         "Satellite" };
-const DefaultStyle satelliteStreets = { "mapbox://styles/mapbox/satellite-streets-v9", "Satellite Streets" };
+const DefaultStyle streets          = { "mapbox://styles/mapbox/streets-v10",           "Streets" };
+const DefaultStyle outdoors         = { "mapbox://styles/mapbox/outdoors-v10",          "Outdoors" };
+const DefaultStyle light            = { "mapbox://styles/mapbox/light-v10",             "Light" };
+const DefaultStyle dark             = { "mapbox://styles/mapbox/dark-v10",              "Dark" };
+const DefaultStyle satellite        = { "mapbox://styles/mapbox/satellite-v10",         "Satellite" };
```

Comment:
I originally authored this PR before any of the v10 styles went public, so it wasn’t clear to me. https://github.com/mapbox/mapbox-gl-native/pull/6301#issuecomment-252031292 tracks other things that need to happen before this PR can land.